### ISSUE TITLE
about m_net output 

### ISSUE THREAD

**Issue by sjf18** [tsing90/pytorch_semantic_human_matting#33]
Hi, thanks for your great work.
I have a question, when i'm training m_net, why I got the output like this
 
![111](https://user-images.githubusercontent.com/30466896/67266833-b270c800-f4e3-11e9-88a3-a6a4f23e3d4a.png)

here is my alpha label and input trimap.

![2](https://user-images.githubusercontent.com/30466896/67266893-cf0d0000-f4e3-11e9-9abd-fc1eab10e501.png)

![3](https://user-images.githubusercontent.com/30466896/67266906-d7653b00-f4e3-11e9-87ab-bb0f73d42185.png)

**Comment by sunweixings** [comment]
Do you have the dataset and  pre-trained model，I want to test one image.Thank you very much.

**Comment by ulnay** [comment]
I have the similar issue here, have you found out what cause the problem? thank you!

> Hi, thanks for your great work.
> I have a question, when i'm training m_net, why I got the output like this
> 
> ![111](https://user-images.githubusercontent.com/30466896/67266833-b270c800-f4e3-11e9-88a3-a6a4f23e3d4a.png)
> 
> here is my alpha label and input trimap.
> 
> ![2](https://user-images.githubusercontent.com/30466896/67266893-cf0d0000-f4e3-11e9-9abd-fc1eab10e501.png)
> 
> ![3](https://user-images.githubusercontent.com/30466896/67266906-d7653b00-f4e3-11e9-87ab-bb0f73d42185.png)

**Comment by hengshan123** [comment]
do you find out the reason? I meet the same problem

**Comment by sjf18** [comment]
> do you find out the reason? I meet the same problem

sorry for replying lately, i still dont know why it happens, so i abandon this project, and use this project instead, [https://github.com/lizhengwei1992/Semantic_Human_Matting](url)
but the matting result is still not good because of missing high quality data
